CHANGES.txt

```diff
@@ -8,6 +8,8 @@ Unreleased
 2018/07/04 2.3.1
 ================
 
+ - Enabled ``loadBalanceHosts`` by default.
```

Comment:
Think this changelog entry can be more descriptive: Suggestion:

Change the default value of the connection property `loadBalanceHosts` to `true`, which will randomize the order in which the client connects of the given CrateDB hosts.